analyzer/src/main/kotlin/managers/Npm.kt

```diff
@@ -591,7 +600,9 @@ open class Npm(
             version = version,
             dependencyNames = dependencyNames,
             packageJson = packageJsonFile
-        )
+        ).also {
+            rawModuleInfoCache[cacheKey] = it
```

Comment:
Can we rather leverage `getOrPut()` here?

Reply:
There is no reading involved: `get` would always return `null` so the "or put" code branch would always be executed. I'd find this a bit misleading. Ok if we keep it as-is?

Reply:
I'm confused, doesn't `rawModuleInfoCache[cacheKey]?.let { return it }` in line 571 do the `get`?

Reply:
Ah, you want all the code go into the getOrPut lambda. That wasn't clear to me...

Reply:
...yes, as anything else wouldn't make sense 😉